### PR TITLE
[aon_timer] Basic DV smoke tests

### PR DESCRIPTION
**Pull request by tjaychen**
- Add basic DV tb that only tests CSR.

### CODE REVIEW

**Comment by tjaychen** [comment]
would the DV guys mind double checking if i added the basic tb correctly?

**Comment by sriyerg** [comment]
> would the DV guys mind double checking if i added the basic tb correctly?

If CSR tests are running and passing, you have done it correctly :)
You could add it to the CI / nightly also by adding it to `hw/top_earlgrey/dv/top_earlgrey_sim_cfg.hjson`.

File: hw/ip/aon_timer/dv/aon_timer_sim.core [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+targets:
+  sim: &sim_target
+    toplevel: tb
+    filesets:
+      - files_rtl
+      - files_dv
+    default_tool: vcs
+
+  # TODO: add a lint check cfg in `hw/top_earlgrey/lint/top_earlgrey_dv_lint_cfgs.hjson`
```
Comment (sriyerg) [inline hw/ip/aon_timer/dv/aon_timer_sim.core:28]: I think this should be easy to finish off right away.

File: hw/ip/aon_timer/dv/aon_timer_sim_cfg.hjson [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    // TODO: add more tests here
+  ]
+
+  // List of regressions.
+  regressions: [
+    {
+      name: smoke
+      tests: ["aon_timer_smoke"]
```
Comment (sriyerg) [inline hw/ip/aon_timer/dv/aon_timer_sim_cfg.hjson:61]: If you comment this out for now, you can add aon_timer to CI / nightly.

**Comment by weicaiyang** [comment]
> > would the DV guys mind double checking if i added the basic tb correctly?
> 
> If CSR tests are running and passing, you have done it correctly :)
> You could add it to the CI / nightly also by adding it to `hw/top_earlgrey/dv/top_earlgrey_sim_cfg.hjson`.

smoke_seq isn't completed yet, so it's not ready for CI.

**Review by weicaiyang — APPROVED** [review]
LGTM, thanks

**Review by sriyerg — APPROVED** [review]

**Comment by sriyerg** [comment]
@tjaychen, CI is complaining about the missing testplan. `uvmdvgen` actually generates not just the `dv` but `data` and `doc` too. Would you please check? It will be good to add the biolerplate doc as well which is also missing.

**Pull request merged**